operations/node_management/Add_Remove_Replace_NCNs/Validate_NCN.md

```diff
@@ -0,0 +1,207 @@
+# Validate Added NCN
+
+Only follow the steps in the section for the node type that was added:
+
+- [Master Node](#validate-master-node)
+- [Worker Node](#validate-worker-node)
+- [Storage Node](#validate-storage-node)
+
+<a name="validate-master-node"></a>
+
+## Validate: Master Node
+
+Validate the master node added successfully.
+
+1. Verify the new node is in the cluster.
+
+    Run the following command from any master or worker node that is already in the cluster. It is helpful to run this command several times to watch for the newly rebuilt node to join the cluster. This should occur within 10 to 20 minutes.
+
+    ```bash
+    ncn-mw# kubectl get nodes
+    ```
+
+    Example output:
+
+    ```text
+    NAME       STATUS   ROLES    AGE    VERSION
+    ncn-m001   Ready    master   113m   v1.19.9
+    ncn-m002   Ready    master   113m   v1.19.9
+    ncn-m003   Ready    master   112m   v1.19.9
+    ncn-w001   Ready    <none>   112m   v1.19.9
+    ncn-w002   Ready    <none>   112m   v1.19.9
+    ncn-w003   Ready    <none>   112m   v1.19.9
+    ```
+
+1. Confirm the `sdc` disk has the correct `lvm` on the rebuilt node.
```

Comment:
I thought this wasn't always `sdc`. I remember the Goss test which checked for it had to be modified so it wasn't hard-coding the device name. I thought the important thing was that the label was showing up on some disk, but it didn't matter which one.

For master nodes, the Goss test runs `blkid -L ETCDLVM` and verifies that the command passes.

Also, this step should specify where the command is to be run.

Reply:
Good catch - fixed